client/x11/Makefile.am

```diff
@@ -67,6 +67,13 @@ noinst_HEADERS = \
 	locales.h \
 	$(NULL)
 
+xwaylandsessiond_in_files = 10-ibus-x11.in
+xwaylandsessiond_DATA = $(xwaylandsessiond_in_files:.in=)
+xwaylandsessionddir=$(sysconfdir)/xdg/Xwayland-session.d
+
+10-ibus-x11: 10-ibus-x11.in
+	$(AM_V_GEN) sed -e "s|\@libexecdir\@|$(libexecdir)|"  $< > $@.tmp && mv $@.tmp $@
```

Comment:
Please add `chmod 755 $@`

Reply:
Good point. That was still not enough to make the script at /etc/xdg/Xwayland-session.d executable, so I added a `install-data-hook` to ensure it ends up executable.